docs/understanding-radicle/faq.md

```diff
@@ -1,14 +1,135 @@
+isn't a single `master` branch that contributors merge into. Each peer maintains
+a view of a project with their changesets and branches. These views are gossiped
+around to other peers that are interested in those changes. Read more about the
+implications and approach to this design [here][sm].
+
+## How is Radicle more secure than centralized platforms?
+The Radicle network is peer-to-peer and built on public key cryptography. To
+start, this means that there is no need to rely on third parties to access or
+use the Radicle network. It is harder to take down because there is no central
+point of failure, and is resistant to corporate capture and censorship. In
+addition, all data on the Radicle network is cryptographically signed & verified
+as it's gossiped between peers. While centralized platforms rely on user
+interface components and key oracles to signal trust from user to user, Radicle
+has designed trust into the core of the protocol.
+
+## How will issues and PRs work? 
+Social collaboration features (i.e. bug reports, patches, discussions etc...)
+are all on the Radicle roadmap. They will work very similarly to the experiences
+we have now, but will be local-first and cryptographically signed. This means
+issues, PRs, and discussions will be more secure, available offline, and stored
+on your machine — not on a central server!
+
+## When CLI tooling?
+We're working on it! We will introduce CLI tooling alongside of Upstream
+development.
+
+## Can I backup a GitHub project on Radicle?
+Yes! Publishing a codebase to Radicle is a great way to create a peer-to-peer
+backup of your repositories. Maintaining a mirror of a project on Radicle is as
+simple as pushing to another remote. Read more about [creating and sharing
+projects][cp].
+
+## Can I replace GitHub with Radicle?
+If you want! While our Beta release will have the basic collaboration features
+(i.e. code hosting, sharing, checking out, and pushing/pulling), we plan to
+introduce features that could support a similar day-to-day code collaboration
+experience to GitHub. They will include bug reporting, patches, code review, and
+discussions. 
+
+That being said, while we believe that reducing one's reliance on
+centrally-hosted platforms is generally a good idea, we also believe that code
+collaboration solutions serve different purposes for different people. Radicle
+Upstream *will* support social collaboration, but it's priority will be
+delivering secure, local-first, peer-to-peer code collaboration — not an exact
+GitHub replica. 
+
+## Where is my data stored? 
+On the Radicle network, content is distributed peer-to-peer via a process called
+gossip. This means that peers self-host their own content — and the content of
+any peers they are interested in — locally on their machine. It also means that
+whenever your data is published to the network, it can be replicated and stored
+on another peer's machine.
+
+## Can I create private repositories on Radicle?
```

Comment:
Is this true, will we have private projects in the future? I thought you could have private projects today by encrypting you repo and storing that, maybe I misunderstood.

I wonder if it's worth mentioning that everything you put on radicle will be publicly available and to be wary about that.

Reply:
Will add the disclaimer. @cloudhead said that we can have private projects in the future, but I don't think we can have them now. He can clarify.

Reply:
There isn't any support for encrypted projects right now.